ui/components/app/transaction-alerts/transaction-alerts.js

```diff
@@ -30,74 +34,61 @@ const TransactionAlerts = ({
-              align="left"
-              className="transaction-alerts__pending-transactions"
-              margin={0}
-              tag={TypographyVariant.paragraph}
-              variant={TypographyVariant.H7}
-            >
-              <strong>
-                {pendingTransactions?.length === 1
-                  ? t('pendingTransactionSingle', [pendingTransactions?.length])
-                  : t('pendingTransactionMultiple', [
-                      pendingTransactions?.length,
-                    ])}
-              </strong>{' '}
-              {t('pendingTransactionInfo')}
-              {t('learnCancelSpeeedup', [
-                <a
-                  key="cancelSpeedUpInfo"
-                  href={ZENDESK_URLS.SPEEDUP_CANCEL}
-                  rel="noopener noreferrer"
-                  target="_blank"
-                >
-                  {t('cancelSpeedUp')}
-                </a>,
-              ])}
-            </Typography>
-          }
-          useIcon
-          iconFillColor="var(--color-warning-default)"
-          type="warning"
-        />
+        <BannerAlert severity={SEVERITIES.WARNING}>
+          <Typography
+            align="left"
+            className="transaction-alerts__pending-transactions"
+            margin={0}
+            tag={TypographyVariant.paragraph}
+            variant={TypographyVariant.H7}
+          >
+            <strong>
+              {pendingTransactions?.length === 1
+                ? t('pendingTransactionSingle', [pendingTransactions?.length])
+                : t('pendingTransactionMultiple', [
+                    pendingTransactions?.length,
+                  ])}
+            </strong>{' '}
+            {t('pendingTransactionInfo')}
+            {t('learnCancelSpeeedup', [
+              <a
+                key="cancelSpeedUpInfo"
+                href={ZENDESK_URLS.SPEEDUP_CANCEL}
+                rel="noopener noreferrer"
+                target="_blank"
+              >
+                {t('cancelSpeedUp')}
+              </a>,
+            ])}
+          </Typography>
+        </BannerAlert>
       )}
       {estimateUsed === PriorityLevels.low && (
-        <ActionableMessage
-          dataTestId="low-gas-fee-alert"
-          message={
-            <Typography
-              align="left"
-              margin={0}
-              tag={TypographyVariant.paragraph}
-              variant={TypographyVariant.H7}
-            >
-              {t('lowPriorityMessage')}
-            </Typography>
-          }
-          useIcon
-          iconFillColor="var(--color-warning-default)"
-          type="warning"
-        />
+        <BannerAlert
+          data-testid="low-gas-fee-alert"
+          severity={SEVERITIES.WARNING}
+        >
+          <Typography
+            align="left"
+            margin={0}
+            tag={TypographyVariant.paragraph}
+            variant={TypographyVariant.H7}
+          >
+            {t('lowPriorityMessage')}
+          </Typography>
```

Comment:
I think we can remove the Typography component all together?
```suggestion
          {t('lowPriorityMessage')}
```

Reply:
Cool, I'll remove those blocks and see how it looks. I was originally thinking of adding stories too, just got complicated with state - I'll look into it again today and try figure it out.

Reply:
Do a search for `configureStore` and only accept `*.stories.js` it should give you some examples

Reply:
ok, thanks -  I was able to get 3 of the four alerts to show by manipulating state. I'll work on getting the pending transactions alert state tomorrow.

![image](https://user-images.githubusercontent.com/82925669/223332860-c54c0c8e-8027-42cb-94e6-64a2a016f33f.png)

Reply:
Pending transactions alert is now in, with separated stories for each alert scenario

![image](https://user-images.githubusercontent.com/82925669/223588549-9d00cb73-e867-4167-aef4-b0f06fc6dda1.png)